lib/common/ignore_not_found.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+module Common
```

Comment:
minor but maybe this can go in `controllers/concerns` since it's controller specific?